tools/src/commands/info_command.cc

```diff
@@ -133,10 +135,45 @@ void InfoCommand::print_tile_sizes() const {
   auto attributes = schema.attributes();
   uint64_t total_persisted_size = 0, total_in_memory_size = 0;
 
+  struct tile_size_extremes_t {
```

Comment:
Figured short-term, given story that originally sought a tool (rather than an API), since they were done, to go ahead and include.  It may be faster for those individuals wanting to evaluate some existing items for ball-parking sizes in current arrays than having to code/test an application to use the API to obtain that ball-parking information.

Reply:
@Shelnutt2  thoughts?

Reply:
@dhoke4tdb on second thought, unless we can test this CLI usage, then please pull out (and save) this code, and defer for now.

Reply:
reverted